app/index.js

```diff
@@ -1,8 +1,8 @@
-const generators = require('yeoman-generator')
+const Generator = require('yeoman-generator')
 const humps = require('humps')
 
-module.exports = generators.Base.extend({
-  prompting: function() {
+module.exports = class extends Generator {
```

Comment:
Should this class have a name for debug-ability/stack traces?  Not sure if we'd ever see it.